content/en/docs/configuring-sdk/custom-http.md

```diff
@@ -40,7 +73,41 @@ See <https://golang.org/pkg/net/#Dialer.Timeout>
 
 We'll call this ``Connect`` as **time.Duration**.
 
-## Transport.ExpectContinueTimeout
+## Transport
+
+The `BuildableHTTPClient` provides a builder mechanics for constructing clients with modified 
+[Transport](https://golang.org/pkg/net/http#Transport) options.
+
+### Configuring a Proxy
+
+If you cannot directly connect to the internet, you can use Go-supported
+environment variables (`HTTP_PROXY` / `HTTPS_PROXY`) or create a custom HTTP client to
+configure your proxy. The following example will configure the client to use `PROXY_URL` as the proxy
+endpoint:
+
+```go
+httpClient := aws.NewBuildableHTTPClient().WithTransportOptions(func(tr *http.Transport) {
+	proxyURL, err := url.Parse("PROXY_URL")
+	if err != nil {
+		log.Fatal(err)
+	}
+	tr.Proxy = http.ProxyURL(proxyURL)
+})
+```
+
+### Other Settings
+
+Below are other `Transport` settings that can be modified to tune the HTTP client. These settings can be applied
+as shown in the example below:
+
+```go
+httpClient := aws.NewBuildableHTTPClient().WithTransportOptions(func(tr *http.Transport) {
+	tr.ExpectContinueTimeout = 0
+	tr.MaxIdleConns = 10
+})
+```
```

Comment:
Maybe helpful to link to the go doc for the transport instead to arbitrary fields?

Reply:
Each of these specific sections has a link to the Go doc, are you just suggesting we don't have our own flavor text and just link to specific ones we want to call out?